tests/unit/moduleapi/cluster.tcl

```diff
@@ -22,6 +22,24 @@ start_cluster 3 0 [list config_lines $modules] {
             fail "node 2 or node 3 didn't receive cluster module message"
         }
     }
+
+    $node1 CONFIG RESETSTAT
+    $node2 CONFIG RESETSTAT
+    $node3 CONFIG RESETSTAT
+
+    test "Cluster module message DING/DONG acknowledgment" {
+        assert_equal OK [$node1 test.pingall]
+        wait_for_condition 50 100 {
+            [CI 0 cluster_stats_messages_module_received] eq 2  &&
+            [CI 1 cluster_stats_messages_module_received] eq 1 &&
+            [CI 2 cluster_stats_messages_module_received] eq 1
+        } else {
+            fail "node 2 or node 3 didn't receive DING messages or node 1 didn't receive DONG message"
+        }
+
+        verify_log_message -1 "*DING (type 1) RECEIVED*Hey*" 0
+        verify_log_message 0 "*DONG (type 2) RECEIVED*Message Received*" 0
```

Comment:
As both receiver will send back `DONG`, we should use the utility method `count_log_message` to verify if we received `DONG` twice.